bin/kn

```diff
@@ -124,6 +136,8 @@ while [[ "$1" == -* ]]; do
   esac
 done
 
+echo "Using provisioner version: $KN_PROVISIONERS_IMG"
```

Comment:
Part of the possibility to have different provisioners image depenting on if you are on a development branch.